src/autoscaler/scalingengine/config/config_test.go

```diff
@@ -102,6 +109,7 @@ cf:
 db:
   policy_db_url: test-policy-db-url
   scalingengine_db_url: test-scalingengine-db-url
+  SchedulerDbUrl: test-scheduler-db-url
```

Comment:
Upper case?!?
This is working because Go does case-insensitive matching for you.